apis/PackageManager/IAppPackageManager.h

```diff
@@ -232,7 +232,7 @@ namespace Exchange {
 
         // @brief ListPackages
         // @text listPackages
-        virtual Core::hresult ListPackages(IPackageIterator*& packages /* @out */) = 0;
+        virtual Core::hresult ListPackages(string& packagesJson /* @out @opaque */) = 0;
```

Comment:
Usage of "opaque" is discouraged in COMRPC. Consider fixing the API Spec in such a way that the result object is an array.